packs/data/classfeatures.db/animal-instinct.json

```diff
@@ -23,51 +23,393 @@
         },
         "rules": [
             {
-                "adjustName": false,
-                "allowedDrops": {
-                    "label": "level 1 barbarian class feature",
-                    "predicate": [
-                        "item:level:1",
-                        "item:trait:barbarian",
-                        "item:type:feature"
-                    ]
-                },
                 "choices": [
                     {
-                        "value": "Compendium.pf2e.classfeatures.Ape Animal Instinct"
+                        "label": "PF2E.SpecificRule.Barbarian.Animal.Ape",
+                        "value": "ape"
                     },
                     {
-                        "value": "Compendium.pf2e.classfeatures.Bear Animal Instinct"
+                        "label": "PF2E.SpecificRule.Barbarian.Animal.Bear",
+                        "value": "bear"
                     },
                     {
-                        "value": "Compendium.pf2e.classfeatures.Bull Animal Instinct"
+                        "label": "PF2E.SpecificRule.Barbarian.Animal.Bull",
+                        "value": "bull"
                     },
                     {
-                        "value": "Compendium.pf2e.classfeatures.Cat Animal Instinct"
+                        "label": "PF2E.SpecificRule.Barbarian.Animal.Cat",
+                        "value": "cat"
                     },
                     {
-                        "value": "Compendium.pf2e.classfeatures.Deer Animal Instinct"
+                        "label": "PF2E.SpecificRule.Barbarian.Animal.Deer",
+                        "value": "deer"
                     },
                     {
-                        "value": "Compendium.pf2e.classfeatures.Frog Animal Instinct"
+                        "label": "PF2E.SpecificRule.Barbarian.Animal.Frog",
+                        "value": "frog"
                     },
                     {
-                        "value": "Compendium.pf2e.classfeatures.Shark Animal Instinct"
+                        "label": "PF2E.SpecificRule.Barbarian.Animal.Shark",
+                        "value": "shark"
                     },
                     {
-                        "value": "Compendium.pf2e.classfeatures.Snake Animal Instinct"
+                        "label": "PF2E.SpecificRule.Barbarian.Animal.Snake",
+                        "value": "snake"
                     },
                     {
-                        "value": "Compendium.pf2e.classfeatures.Wolf Animal Instinct"
+                        "label": "PF2E.SpecificRule.Barbarian.Animal.Wolf",
+                        "value": "wolf"
                     }
                 ],
-                "flag": "animal",
                 "key": "ChoiceSet",
-                "prompt": "PF2E.SpecificRule.Barbarian.Animal.Prompt"
+                "prompt": "PF2E.SpecificRule.Barbarian.Animal.Prompt",
+                "rollOption": "feature:instinct:animal"
```

Comment:
Can we switch this to `animal-instinct`? That makes it look like a class feature item, and I'd rather not risk that biting us in the future.

Reply:
Done